tests/utils/nodes/nodes.go

```diff
@@ -39,3 +39,17 @@ func IsNodeInCondition(node *corev1.Node, condition corev1.NodeConditionType) bo
 
 	return false
 }
+
+// GetNumOfReadyNodesInCluster get the number of ready nodes in the cluster.
+func GetNumOfReadyNodesInCluster(cs *client.ClientSet) int32 {
+	nodesList, _ := cs.Nodes().List(context.Background(), metav1.ListOptions{})
```

Comment:
Why is error ignored? You need to process error on the level above. Please refactor this func
`GetNumOfReadyNodesInCluster(cs *client.ClientSet) (int32, error)`